extensions/applicationinsights-clickanalytics-js/src/Interfaces/Datamodel.ts

```diff
@@ -0,0 +1,382 @@
+/**
+ * DataModel.ts
+ * @author Krishna Yalamanchili(kryalama) and Hector Hernandez(hectorh)
+ * @copyright Microsoft 2020
+ * File containing the interfaces for Web Analytics SDK.
+ */
+
+
+import { EventType } from '../Enums';
+
+export const DEFAULT_DONOT_TRACK_TAG = 'ai-dnt';
```

Comment:
nit: These constants are only used by a single internal file, maybe put these in that file